README.md

```diff
@@ -194,23 +188,23 @@ We gladly credit all contributors.
   these companies are usually formed in Delaware, so legalities of all this are defined in
   Delaware law.
 
-### What is compensation?
-
-🄳 **Compensation** is what you get in exchange for working for a company.
-Some compensation is cash—your salary and bonus, if you get one.
-Other parts are benefits, like health insurance, family-related protections, perks, and
-retirement plans.
-But for many jobs, compensation includes a kind of ownership in a company, called equity.
-It’s this last part we focus on.
-
-🚧 Define equity compensation formally and also distinguish or discuss “total rewards.”
+### Public and private companies
 
-To understand exactly how compensation works, it helps to start at the beginning:
-What is a company, exactly?
-What is ownership, exactly?
-And what is this ownership worth?
+- 🄳 [**Public companies**](https://en.wikipedia.org/wiki/Public_company) are corporations where
+  any member of the public can own stock.
+  People can buy and sell the stock for cash on public
+  [exchanges](https://www.investopedia.com/terms/e/exchange.asp). The value of a company’s
+  shares is the value you see in the stock market reports, so shareholders know how much
+  their stock is worth.
+- 🄳 Most smaller companies, including all startups, are
+  [**private companies**](https://en.wikipedia.org/wiki/Privately_held_company) with owners who
+  control how those companies operate.
+  Unlike a public company, where anyone is able to buy and sell stock, owners of a private
+  company control who is able to buy and sell their stock.
+  There may be few or no transactions, or they may not be publicly known.
+- 🚧 What are public exchanges and how stock is bought and sold in practice?
 
-### What is stock and what is equity?
+### What is stock?
 
 - 🄳 [**Stock**](https://en.wikipedia.org/wiki/Stock) is a legal invention that helps track
```

Comment:
Missing from either Stock DO or Equity DO why "ownership in a company" matters. Do I get a vote on company policy? Will someone ever ask my opinion 'as a part owner'? Do I get a share in the company's revenue? Why should I care about ownership?